### PR TITLE
Fix: Icon List - Text size increased on Icon list widget with size set to `em` and a link [ED-3177]

### PR DESCRIPTION
**Pull request by guyc-E**
…t to `em` and a link [ED-3177]

## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

[ED-3177]: https://elementor.atlassian.net/browse/ED-3177

### CODE REVIEW

File: includes/widgets/icon-list.php [hunk 1]
```diff
@@ -522,7 +522,7 @@ protected function register_controls() {
 			Group_Control_Typography::get_type(),
 			[
 				'name' => 'icon_typography',
-				'selector' => '{{WRAPPER}} .elementor-icon-list-item, {{WRAPPER}} .elementor-icon-list-item a',
+				'selector' => '{{WRAPPER}} .elementor-icon-list-item > .elementor-icon-list-text, {{WRAPPER}} .elementor-icon-list-item > a',
```
Comment (rotemee) [inline includes/widgets/icon-list.php:525]: The list item font-size is still affected when creating a new widget and setting a global link font-size.

Comment (guyc-E) [reply includes/widgets/icon-list.php:525]: Fixed

File: assets/dev/scss/frontend/widgets/icon-list.scss [hunk 2]
```diff
@@ -57,7 +57,8 @@
 
 		&, a {
 			display: flex;
-			align-items: flex-start;
+			align-items: center;
+			font-size:inherit;
```
Comment (rotemee) [inline assets/dev/scss/frontend/widgets/icon-list.scss:61]: Add a space before the inherit.

File: includes/widgets/icon-list.php (hunk 1 above)
Comment (rotemee) [reply includes/widgets/icon-list.php:525]: The bug still happens - STR:
1. Set a global link font-size.
2. Drag a new icon-list widget.
3. Add a link url to the first list item.
4. The first item's font-size will be larger than the rest.

Comment (guyc-E) [reply includes/widgets/icon-list.php:525]: It works good for me.
Are you sure your grunt was activated?
Can you try again and if it's not working for you let's do it together.

File: assets/dev/scss/frontend/widgets/icon-list.scss (hunk 2 above)
Comment (guyc-E) [reply assets/dev/scss/frontend/widgets/icon-list.scss:61]: done

**Review by rotemee — APPROVED** [review]

File: includes/widgets/icon-list.php (hunk 1 above)
Comment (rotemee) [reply includes/widgets/icon-list.php:525]: Confirmed.

**Pull request merged**